client/src/test/java/io/pravega/client/tables/impl/KeyValueTableTestBase.java

```diff
@@ -44,6 +44,7 @@
      */
     @Test
     public void testSingleKeyConditionalInserts() {
+        Assert.assertTrue(isScopeCreated);
```

Comment:
What's the point of this? Do you not do this in the setup?

Reply:
This is to make sure ScopeCreation has succeeded, before attempting Table creation so that we do not get a SCOPE_NOT_FOUND...

Reply:
So you'll either get SCOPE_NOT_FOUND or the test will fail due to this assertion. The outcome is the same. That's why I asked what the point is.

Reply:
Yes... either ways the test would fail if scope is not created...